README.md

```diff
@@ -1,70 +1,53 @@
-# Getting Started with Create React App
+# Frontend Socialmedia app: PenFriend
 
```

Comment:
Maybe add in a `[![Contributors][contributors-shield]][contributors-url]`

Reply:
måste läggas till sökvägen för dessa i botten av ReadMe'n

Reply:
added